### PR TITLE
Add knative-serving-operator to relatedImages in 1.4.1 CSV

### PR DESCRIPTION
**Pull request by warrenvw**
It was missed in 7df03b2.

### CODE REVIEW

File: olm-catalog/serverless-operator/1.4.1/serverless-operator.v1.4.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -445,5 +445,7 @@ spec:
       image: $IMAGE_KNATIVE_OPERATOR
     - name: knative-openshift-ingress
       image: $IMAGE_KNATIVE_OPENSHIFT_INGRESS
+    - name: knative-serving-operator
+      image: registry.svc.ci.openshift.org/openshift/knative-v0.11.1:knative-serving-operator
```
Comment (warrenvw) [inline olm-catalog/serverless-operator/1.4.1/serverless-operator.v1.4.1.clusterserviceversion.yaml]: @jcrossley3 Should I replace the above k-s-o image with `$IMAGE_KNATIVE_SERVING_OPERATOR`?

Comment (warrenvw) [reply olm-catalog/serverless-operator/1.4.1/serverless-operator.v1.4.1.clusterserviceversion.yaml]: ... with a corresponding change to `hack/catalog.sh`?

Comment (markusthoemmes) [reply olm-catalog/serverless-operator/1.4.1/serverless-operator.v1.4.1.clusterserviceversion.yaml]: Hmm, negative. That image comes "from the outside" and is not built by this repo.

**Pull request merged**